docker-compose.yml

```diff
@@ -23,6 +23,13 @@ services:
       - "14250:14250"
       - "14268:14268"
       - "16686:16686"
+    # This dependency is to insure that localstack is fully initialized and the test bucket
+    # created before docker-compose considers the stack fully up. Without this the localstack
+    # image will be labeled ok with indeterminate latency in bucket creation because the
+    # init script to create the bucket runs in the background.
+    depends_on:
```

Comment:
Docker Compose introduced a `--wait` option that waits for all the services to be healthy. I've just tested it local and it seems to work. We can remove this `depends_on` and the option in Makefile for the `docker-compose-up` command.

Reply:
Yeah, that's a much better solution but of course things are never simple. It looks like docker-compose is now a CLI plugin which is only available as `docker compose` by default on a Mac. The `docker-compose` I had was V1 and I didn't realize it wasn't being updated by Docker Desktop. To get a wrapper to enable `docker-compose` to be based on V2 you need to brew install it separately. 

Is `docker compose` normally available on Linux? If so it may make sense to disambiguate and use that in the Makefile along with `--wait` since that's the future trajectory for this command.

Reply:
That's just me obviously but `docker compose` seems to work on my machine (Manjaro):
```bash
guilload@x13 ~> docker compose version
Docker Compose version 2.6.1
```

Reply:
OK, changed to use `docker compose` and `--wait`. There is also significant latency on the startup of some of the other services before their health checks pass, especially Kafka.